modules/home/assets/js/components/site-settings-list-item.js

```diff
@@ -0,0 +1,89 @@
+import ListItem from '@elementor/ui/ListItem';
+import ListItemText from '@elementor/ui/ListItemText';
+import Link from '@elementor/ui/Link';
+import Dialog from '@elementor/ui/Dialog';
+import Button from '@elementor/ui/Button';
+import DialogActions from '@elementor/ui/DialogActions';
+import DialogContent from '@elementor/ui/DialogContent';
+import DialogContentText from '@elementor/ui/DialogContentText';
+import DialogHeader from '@elementor/ui/DialogHeader';
+import DialogHeaderGroup from '@elementor/ui/DialogHeaderGroup';
+import DialogTitle from '@elementor/ui/DialogTitle';
+import Box from '@elementor/ui/Box';
+import TextField from '@elementor/ui/TextField';
+
+const SiteSettingsListItem = ( { item, image } ) => {
+	const [ open, setOpen ] = React.useState( false );
+	const [ pageName, setPageName ] = React.useState( '' );
+
+	const handlePopupOpen = ( e ) => {
```

Comment:
Also in other places in this file.

```suggestion
	const handlePopupOpen = ( event ) => {
```